graf3d/eve7/inc/ROOT/REveDataSimpleProxyBuilderTemplate.hxx

```diff
@@ -18,29 +18,29 @@ public:
 
 protected:
    using REveDataSimpleProxyBuilder::Build;
-   virtual void Build(const void *iData, REveElement *itemHolder, const REveViewContext *context)
+   virtual void Build(const void *iData, unsigned int index, REveElement *itemHolder, const REveViewContext *context)
```

Comment:
Do we *really* need `unsigned` here? While it might never be `< 0` in practice, `unsigned` gets us the wrapping semantics of arithmetics that we usually don't want. I'd warmly recommend to simply use `int`!

Reply:
CMS software uses unsigned int, that is the reason it is here. There is no problem changing it to int.